utilities/data/getAllStories.ts

```diff
@@ -1,11 +1,19 @@
 import { type ISbStoriesParams } from '@storyblok/react/rsc';
-import { unstable_cache } from 'next/cache';
+import { cache } from 'react';
```

Comment:
Cache is now stable in React and is the recommended approach over unstable_cache.

Reply:
Hmmm. Yeah, I went with cache as it seemed the quicker swap for unstable cache however, `use cache` is more nextjs-y. I'll move in that direction.